### DISCUSSION TITLE
Allow a user have access to certain domain(s) only?

### DISCUSSION THREAD

**Discussion by naddika** [plausible/analytics#2115] [Anything Else]
Let's say, I have a 1000 websites. 

Is there a way to permit a new user to have access **only** to 1..5 certain domains? That is, by default such a user would have NO access to anything **but** 1..5 domains that I as the admin choose? 

Currently it's the other way around, right? Namely, a user has access to every registered domain **unless** you restrict him to certain ones.

**Comment by metmarkosaric** [comment]
hi @naddika! this is how it works currently. you can invite a team member to any specific site that you wish. see https://plausible.io/docs/users-roles

**Comment by naddika** [comment]
I see, thx
